### PR TITLE
Download file from url

### PR DESCRIPTION
**Pull request by cynepco3hahue**

### CODE REVIEW

File: rrmngmnt/host.py [hunk 1]
```diff
@@ -336,3 +337,41 @@ def is_connective(self, tcp_timeout=20.0):
             ret = False
 
         return ret
+
+    def download_file_from_url(self, url, f_dir):
```
Comment (myakove) [inline rrmngmnt/host.py]: maybe use  https://pypi.python.org/pypi/wget 
and name can be only download_file()

Comment (cynepco3hahue) [inline rrmngmnt/host.py]: I will rename function
About wget module, I believe I has some problem with our framework:
Traceback (most recent call last):
  File "/home/alukiano/PycharmProjects/ART/art/test_handler/plmanagement/plugins/unittest_test_runner_plugin.py", line 209, in __call__
    self.f()
  File "/home/alukiano/PycharmProjects/ART/art/tests/rhevm/unittests/local_test/local_test.py", line 40, in test01_create_vm
    filename = wget.download(url=url, out="/tmp")
  File "/usr/lib/python2.7/site-packages/wget.py", line 526, in download
    (tmpfile, headers) = ulib.urlretrieve(binurl, tmpfile, callback)
  File "/usr/lib64/python2.7/urllib.py", line 98, in urlretrieve
    return opener.retrieve(url, filename, reporthook, data)
  File "/usr/lib64/python2.7/urllib.py", line 271, in retrieve
    reporthook(blocknum, bs, size)
  File "/usr/lib/python2.7/site-packages/wget.py", line 513, in callback_charged
    callback_progress(blocks, block_size, total_size, bar_function=bar)
  File "/usr/lib/python2.7/site-packages/wget.py", line 461, in callback_progress
    width = min(100, get_console_width())
  File "/usr/lib/python2.7/site-packages/wget.py", line 337, in get_console_width
    ioctl(sys.stdout.fileno(), TIOCGWINSZ, winsize)
AttributeError: StringIO instance has no attribute 'fileno'
and I do not want to spend time on investigation)

**Comment by lukas-bednar** [comment]
I have 2 problems with this function.

1) it transfers data over network from A to B and then finally to C.
I want to see transfer from A to C.

2) I want to reduce number of methods to be added to Host class, I don't want to end up with 3k file, and 160 methods as we saw in machine.py module.
Every method needs purpose and specific place to put. as we do with network, filesystem, storage, ...

**Comment by cynepco3hahue** [comment]
1) Not really understand what you mean, I open host session and with session open file on host and write straight forward to this file

2) I can move it, but can you specify where better place for it, because I also want to use it under ART and under jenkins repo

**Comment by lukas-bednar** [comment]
1) there is h1/file.txt , you are on h2 running python script, which read that file into memory of h2 (transfer from A to B). then you copy this data over ssh to h3 where you want to download this file to (transfer from B to C).
when I see this method I imagine that h3 download file from h1 (transfer from A to C).

2) I would put it under filesystem, or you can create separate module for that if you can find dedication or some common base with other methods. search your engineering feelings ;)

**Comment by cynepco3hahue** [comment]
Will open new one

**Pull request closed without merging**